Common/Core/TrackSelectionDefaults.cxx

```diff
@@ -76,4 +79,32 @@ TrackSelection getGlobalTrackSelectionSDD()
   return selectedTracks;
 }
 
+// Default track selection for nuclei analysis in run3 (STILL JUST A PLACEHOLDER)
+TrackSelection getGlobalTrackSelectionRun3Nuclei()
+{
+  return getGlobalTrackSelection();
+}
+
+// Default track selection for HF analysis (global tracks, with its points, but no tight selection for primary) in run3 (STILL JUST A PLACEHOLDER)
+TrackSelection getGlobalTrackSelectionRun3HF()
```

Comment:
Maybe we should start from the `getGlobalTrackSelection` like for nuclei and tweak the differences?